requirements.txt

```diff
@@ -147,8 +147,6 @@ icalendar==5.0.4
     # via -r requirements.in
 idna==3.4
     # via requests
-importlib-metadata==6.0.0
```

Comment:
This should not get removed. You've probably run `pip-compile` with Python=<3.9.

Reply:
Nicely catched 🎣